scipy/spatial/_ckdtree.pyx

```diff
@@ -1546,9 +1545,8 @@ cdef class cKDTree:
 
     def __getstate__(cKDTree self):
         cdef object state
-        cdef np.intp_t size
         cdef ckdtree * cself = self.cself
-        size = cself.tree_buffer.size() * sizeof(ckdtreenode)
+        cdef np.intp_t size = cself.tree_buffer.size() * sizeof(ckdtreenode)  # no-cython-lint
```

Comment:
Just curious: why does this need `# no-cython-lint`?

Reply:
Good eyes ;)

https://github.com/MarcoGorelli/cython-lint/issues/52

Reply:
fixed in version 0.12.1 - thanks for the report, much appreciated!